docs/docs/concepts/jobs.md

```diff
@@ -66,16 +46,60 @@ class JobHandler(JobHandlerInterface):
 - The JobHandlerInterface and Job class are defined in dm-job repository.
 - To access the job entity, use `self.job.entity`
 - - `self.job.entity.applicationInput` is used for supplying inputs to the job. For example, you can include inputs to calculations.
 - - `self.job.entity.outputTarget` is used for defining where the result of the job should be stored.
+- The file must have name `__init__.py`
+
+
+Also, the file must be available as a file in the dm-job container.
+For example, if you want to add a new job to the [dm-core-packages repository](https://github.com/equinor/dm-core-packages), you would create your JobHandler file inside the dm-core-packages repository and make it available for the dm-job API.¨¨
+
+This can be achieved by using docker volumes. 
+For example, in the [docker-compose file in dm-core-packages](https://github.com/equinor/dm-core-packages/blob/main/example/docker-compose.yaml), 
+docker volume is used to "copy" the content of `dm-core-packages/example/job_handlers` into the folder `dm-job/src/job_handler_plugins`.
+
+
+## Job entity
+To run a job, you must define a job entity. Example:
+```json
+{
+    "label": "Example local container job",
+    "type": "dmss://WorkflowDS/Blueprints/Job",
+    "status": "Not started",
+    "triggeredBy": "author",
+    "outputTarget": "dmss://DataSource/$456-456.result",
+    "applicationInput": {
+      "type": "dmss://system/SIMOS/Reference",
+      "referenceType": "link",
+      "address": "dmss://DataSource/$123-123"
+    },
+    "runner": {
+      "type": "dmss://DataSource/apps/MySignalApp/models/SignalGeneratorJob"
+    },
+    "started": "Not started"
+  }
+```
+The most important attributes: 
+- status: what is the status of the job: registered, not starting, running, failed, completed, removed or unknown. 
+- triggeredBy: who started the job
+- outputTarget: defines where to place the result of the job after the job is complete. Normally an address to some entity.
+- applicationInput: is a link reference to another entity that can be of any type (but have to be an object). This is an entity that is sent in with the job, so this entity is directly available  
+inside the start() function in the JobHandler file.
+- runner: Has information needed to run the job. For example what container image to use, environment variables and so on. The type of this attribute should extend from `dmss://WorkflowDS/Blueprints/JobHandler`. 
+- started: the time when the job is started.
 
 
+## Different kinds of jobs
 
-Jobs can be run in different ways:
+Jobs are divided into two main categories, based of how they are executed:
 - locally in the instance / deployment of the dm-job API, either as a "container job" or a "regular job". For container jobs, the job entity's runner type should extend from the  `Container` blueprint from dm-job. For regular jobs, the job entity's runner type should extend from the `JobHandler` blueprint from dm-job.
```

Comment:
So a Container job spins up its own container, right? While the regular job runs in the existing job container?

Reply:
yep!